### PR TITLE
Organize Project to Module: Creating Single-Page Apps

### PR DESCRIPTION
**Pull request by peterdtitan**
This is a repeat of the previous block. The project structure for awesome books has been changed to modules and luxon properties were added. The following files were changed/modified or rewritten: 

- index.js
- luxon.js
- index.html
- modules/ folder to contain all necessary functions and classes.

Screenshot 2022-07-11 at 2 30 21 PM

### CODE REVIEW

File: modules/library.js [hunk 1]
```diff
@@ -51,4 +51,4 @@ export default class Library {
       localStorage.setItem('books', JSON.stringify(this.books));
     }
   }
-}
\ No newline at end of file
+}
```
Comment (julie-ify) [inline modules/library.js]: - [x] Kindly change this function `function storageAvailable(type)` to an ES6 arrow function. hint;

```js
const storageAvailable = (type) => {...}

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @peterdtitan,

Good job so far!

## Highlights!
- GitHub Flow was used ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization and working app ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [x] **[REQUIRED]** Kindly add instructions on how to clone your repository and how to get your app to show on the browser in the Readme. This is to give users instructions on how to successfully run your app.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/library.js (hunk 1 above)
Comment (peterdtitan) [reply modules/library.js]: Function updated 👍🏽

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @peterdtitan,

✔️ Good documentation
✔️ Use of ES6 syntax for functions

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**